src/dtoModule/tileValueDTO.py

```diff
@@ -0,0 +1,20 @@
+
+
+class TileValueDTO():
```

Comment:
what does DTO stand for? Please do not use abbreviations that are not clear and write it out.

Reply:
DataTransfereObject. We use this base classes like Pojos in java.